RELEASE.md

```diff
@@ -78,6 +78,8 @@
 * Removed `RegistrationSpecs` and all registration hooks that belonged to it. Going forward users can register custom library components through `settings.py`.
 * Added the `PluginManager` `hook_manager` argument to `KedroContext` and the `Runner.run()` method, which will be provided by the `KedroSession`.
 * Removed the public method `get_hook_manager()` and replaced its functionality by `_create_hook_manager()`.
+* Added `run_id` and `runner_name` attributes to pipeline hook specs.
```

Comment:
What would a user use the `run_id` for?

Reply:
I think it makes a lot of sense in situations where you are logging runs to another system, we expose it the `node_specs` so I felt it was reasonable to introduce.

Reply:
We had a discussion yesterday where we decided that one session = one run, and so we don't need the `run_id` anymore. I'm in the process of writing a summary of the discussion, but for now see: https://github.com/kedro-org/kedro/pull/1329

Reply:
Thanks! Would it be okay to expose the `session_id`?

Reply:
Actually removed because it's already in `run_params` I was being silly/